accounts/abi/type.go

```diff
@@ -57,16 +58,16 @@ var (
 	// Types can be in the format of:
 	//
 	// 	Input  = Type [ "[" [ Number ] "]" ] Name .
-	// 	Type   = [ "u" ] "int" [ Number ] .
+	// 	Type   = [ "u" ] "int" [ Number ] [ x ] [ Number ].
```

Comment:
I meant that we have now `Type   = [ "u" ] "int" [ Number ] [ x ] [ Number ].`, which has `int` "hard coded" into it. Given that we have different type rules for `int`, `string`, `fixed`, it might be cleaner to describe these three (or more?) type definitions on their own line, rather than a single convoluted one.

e.g.

```
// Input  = Type [ "[" [ Number ] "]" ] Name .
// Type   = [ "u" ] "int" [ Number ] || 
//          "fixed" [[ Number ] [ "x" Number ]] ||
//          "bytes" [ Number ]
//          "string"
```

Note the above is just for the idea, I haven't looked into all the types. Just would be nice to describe all the possible types there.